src/main/kotlin/com/hibob/academy/employeeFeedback/dao/FeedbackDao.kt

```diff
@@ -38,4 +38,34 @@ class FeedbackDao(private val sql: DSLContext) {
             .execute() > 0
     }
 
+
+
+    fun getFeedbackByParams(companyId: Long, params: FeedbackQueryParams): List<FeedbackData> {
```

Comment:
this needs more careful handling. each one of these params might be null. for example, if the employee id is null, the join will cause there to be no results.

please test this